src/main/java/net/sf/jabref/gui/preftabs/FileTab.java

```diff
@@ -285,10 +285,6 @@ public void storeSettings() {
             prefs.put(JabRefPreferences.NON_WRAPPABLE_FIELDS, nonWrappableFields.getText());
             updateSpecialFields = true;
         }
-        // If either of the two last entries were changed, run the update for special field handling:
```

Comment:
See `updateSpecialFieldHandling()` :-) 

It updates `private final Set<String> nonWrappableFields = new HashSet<>(5)` which is never used somewhere else...